### PR TITLE
NO-ISSUE fix sporadic read error in CI when opening controller tar

### PR DESCRIPTION
**Pull request by slaviered**
/assign @YuviGold

### CODE REVIEW

**Review by osherdp — APPROVED** [review]
/lgtm

File: discovery-infra/test_infra/logs_utils.py [hunk 1]
```diff
@@ -54,8 +54,9 @@ def _check_entry_from_extracted_tar(component, tarpath, verify):
             tar.extractall(tempdir)
             extractedfiles = os.listdir(tempdir)
             assert any(component in logfile for logfile in extractedfiles), f'can not find {component} in logs'
-            component_tar = [logfile for logfile in extractedfiles if component in logfile][0]
-            verify(os.path.join(tempdir, component_tar))
+            component_tars = [logfile for logfile in extractedfiles if (component in logfile and logfile.endswith(".gz"))]
+            if len(component_tars) > 0
```
Comment (osherdp) [inline discovery-infra/test_infra/logs_utils.py:58]:
```suggestion
            if len(component_tars) > 0:
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: discovery-infra/test_infra/logs_utils.py (hunk 1 above)
Comment (YuviGold) [reply discovery-infra/test_infra/logs_utils.py:58]:
```suggestion
            if component_tars:
```

**Review by osherdp — APPROVED** [review]
/lgtm

**Review by YuviGold — APPROVED** [review]

**Pull request merged**